pkg/project/file.go

```diff
@@ -27,11 +28,12 @@ func (f File) Detect() (Result, bool, error) {
 
 	lines, err := readFile(fp, 2)
 	if err != nil {
-		return Result{}, false,
-			Err(fmt.Sprintf("error reading file: %s", err))
+		return Result{}, false, Err(fmt.Sprintf("error reading file: %s", err))
 	}
 
-	result := Result{}
+	result := Result{
+		Folder: filepath.Dir(fp),
```

Comment:
I don't think we're running `windows.FormatFilePath` on this before using in `hideProjectFolder`. Let's just call `windows.FormatFilePath` inside `hideProjectFolder` to be sure no matter where the project folder comes from that it's normalized?

Reply:
We are. Take a look at `project.go` line 100.

Reply:
Oh, got it. It runs through `windows.FormatFilePath` no matter which type of project detection is used.

Reply:
Also, we're formatting the `--project-folder` at https://github.com/wakatime/wakatime-cli/pull/615/files#diff-d540dc518660d926557564ec01ffcf0beb6ef055d37b97413412597667850059R90 so both sources of project folder, and Entity are normalized to forward slash. 👍